### PR TITLE
Add links to self-paced scenarios

### PR DESCRIPTION
**Pull request by texasmichelle**
Clarify cluster setup instructions
Remove broken links

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/150)

### CODE REVIEW

File: content/docs/guides/requirements.md [hunk 1]
```diff
@@ -14,12 +14,11 @@ aliases = ["/docs/guides/"]
 * [Tensorflow](https://www.tensorflow.org/get_started/)
 * [Ksonnet](https://ksonnet.io/docs/tutorial)
 
-For an end to end example illustrating in details how to deploy kubeflow and run a training job from scratch, check out this [tutorial](https://ai.intel.com/lets-flow-within-kubeflow/).
+For an end-to-end example illustrating in detail how to deploy kubeflow and run a training job from scratch, check out this [tutorial](https://ai.intel.com/lets-flow-within-kubeflow/).
```
Comment (jlewi) [inline content/docs/guides/requirements.md]: The Intel tutorial is out of date at this point. Do we have something up to date we can link to? Or should we just remove it for now?

File: content/docs/started/getting-started.md [hunk 2]
```diff
@@ -26,11 +26,11 @@ functionality and actively soliciting help from companies and individuals intere
...
 
-This documentation assumes you have a Kubernetes cluster already available.
+This documentation assumes you have a Kubernetes cluster available. If
+not, setup one of these environments first:
 
-  * For local environment try [Minikube setup](/docs/started/getting-started-minikube/).
-  * For cloud environment try [GKE setup](/docs/started/getting-started-gke/).
-  * For general k8s cluster try [quick start](/docs/started/getting-started/#quick-start) or [install via ksonnet](/docs/started/getting-started/#install-via-ksonnet).
+  * Local: [Minikube setup](/docs/started/getting-started-minikube/)
+  * Cloud: [GKE setup](/docs/started/getting-started-gke/)
```
Comment (jlewi) [inline content/docs/started/getting-started.md]: I think we need a 3rd option directing people on what to do if they have an existing cluster that's not minikube/GKE

File: content/docs/guides/requirements.md (hunk 1 above)
Comment (texasmichelle) [reply content/docs/guides/requirements.md]: It doesn't really belong in Requirements and #151 will provide a more up-to-date example, so I'll take it out & reach out to the author about updating his post.

File: content/docs/started/getting-started.md (hunk 2 above)
Comment (texasmichelle) [reply content/docs/started/getting-started.md]: The sentence before these two bullet items assumes an existing k8s cluster. The purpose of these two links is to give people an option if that's not the case.

**Pull request merged**